Description: Use lex/yacc from ply
Forwarded: not-needed
Bug-Debian: http://bugs.debian.org/554613
Author: Jakub Wilk <ubanus@users.sf.net>

Index: b/tools/asn2wrs.py
===================================================================
--- a/tools/asn2wrs.py	2010-02-15 17:23:17.000000000 +0100
+++ b/tools/asn2wrs.py	2010-02-15 17:24:58.000000000 +0100
@@ -47,8 +47,8 @@
 import getopt
 import traceback
 
-import lex
-import yacc
+from ply import lex
+from ply import yacc
 
 # OID name -> number conversion table
 oid_names = {
