Description: Suggest solutions on the missing MIBs popup window.
Forwarded: not-needed
Bug-Debian: http://bugs.debian.org/560727
Author: Balint Reczey <balint@balintreczey.hu>

Index: b/epan/oids.c
===================================================================
--- a/epan/oids.c	2010-02-15 17:24:58.000000000 +0100
+++ b/epan/oids.c	2010-02-15 17:24:58.000000000 +0100
@@ -571,7 +571,10 @@
 
 	if (smi_errors->len) {
 		report_failure("The following errors were found while loading the MIBS:\n%s\n\n"
-					   "The Current Path is: %s\n" , smi_errors->str , path_str);
+					   "The Current Path is: %s\n\nYou can avoid this error message "
+					   "by removing the missing MIB modules at Edit -> Preferences"
+					   " -> Name Resolution -> SMI (MIB and PIB) modules or by "
+					   "installing them using the snmp-mibs-downloader package.\n" , smi_errors->str , path_str);
 		D(1,("Errors while loading:\n%s\n",smi_errors->str));
 	}
 
