Description: Change Icon and Categories for desktop file
Forwarded: not-needed
Author: <fred@candle>

Index: b/wireshark.desktop
===================================================================
--- a/wireshark.desktop	2010-02-15 17:23:17.000000000 +0100
+++ b/wireshark.desktop	2010-02-15 17:24:59.000000000 +0100
@@ -68,10 +68,11 @@
 Comment[fi]=Verkkoliikenne analysaattori
 Comment[fr]=Analyseur de trafic réseau
 Comment[sv]=Nätverkstrafikanalysator
-Icon=wireshark.png
+Icon=hi48-app-wireshark.png
 TryExec=wireshark
 Exec=wireshark
 Terminal=false
 # Categorie entry according to version 1.0 of
 # http://www.freedesktop.org/wiki/Specifications/menu-spec
-Categories=System;Monitor;GTK;
+Categories=GNOME;Network;
+Encoding=UTF-8
